code/modules/mob/living/simple_animal/abnormality/_auxiliary_modes/joke/aleph/sukuna.dm

```diff
@@ -0,0 +1,247 @@
+#define MALEVOLENT_SHRINE_COOLDOWN (60 SECONDS)
+/mob/living/simple_animal/hostile/abnormality/sukuna
+	name = "Sukuna"
+	desc = "The Heian Era asspuller himself."
+	icon = 'ModularTegustation/Teguicons/tegumobs.dmi'
+	icon_state = "sukunad"
+	icon_living = "sukunad"
+	portrait = "sukuna"
+	del_on_death = TRUE
```

Comment:
```suggestion
	del_on_death = FALSE
```
Fairly certain this needs to be set to FALSE or the death() proc code will not run.